src/transport_ofi.h

```diff
@@ -492,23 +543,22 @@ void shmem_transport_put_nb(shmem_transport_ctx_t* ctx, void *target, const void
 
         shmem_transport_put_small(ctx, target, source, len, pe);
 
-    } else if (len <= shmem_transport_ofi_bounce_buffer_size &&
-               ctx->options & SHMEMX_CTX_BOUNCE_BUFFER) {
+    } else if (len <= shmem_transport_ofi_bounce_buffer_size && ctx->cq_ep) {
```

Comment:
I see that checking for a non-null `cq_ep` is equivalent (and probably fewer cycles?) than checking the options, but it's ever so slightly less clear what's being checked.  No problem from me, just thought I'd mention it.

Reply:
Yeah, I went back and forth on whether to clear ```SHMEMX_CTX_BOUNCE``` from options or to leave it at what was requested by the user.  Ultimately, I cleared it, but I left this new check since it should eliminate an instruction.